### PR TITLE
birdc: add page

### PR DESCRIPTION
**Pull request by marek22k**
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 2.0.10

### CODE REVIEW

File: pages/common/birdc.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# birdc
+
+> Bird remote control.
+> Command tool to retrieve information like routes from bird and perform configurations during runtime.
+> More information: <https://bird.network.cz/>.
```
Comment (mfrw) [inline pages/common/birdc.md:5]: NIT:
```suggestion
> More information: <https://bird.network.cz>.
```

File: pages/common/birdc.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+`birdc "show route where bgp_path ~ [{{4242120045}}]"`
+
+- Show all best routes:
+
+`birdc show route primary`
+
+- Show all details of all routes from a given prefix:
+
+`birdc show route for fd00:/8 all`
```
Comment (mfrw) [inline pages/common/birdc.md:37]: We should tokenize:
```suggestion
`birdc show route for {{fd00:/8}} all`
```

File: pages/common/birdc.md (hunk 1 above)
Comment (marek22k) [reply pages/common/birdc.md:5]: Is there a particular reason for this? In other pages (for example `tshark`) this is not the case either.

File: pages/common/birdc.md (hunk 2 above)
Comment (marek22k) [reply pages/common/birdc.md]: Thanks for the hint!

File: pages/common/birdc.md (hunk 1 above)
Comment (mfrw) [reply pages/common/birdc.md:5]: We try to maintain a standard. Note that this is pointed as a nitpick and not something as a blocker.
The idea is to follow something, e.g: - do we have a trailing `/` or not.
As per my opinion, the consensus has been to drop the trailing `/`.

**Review by mfrw — APPROVED** [review]
LGTM :)

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/birdc.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+`birdc show status`
+
+- Show all active protocols:
+
+`birdc show protocols`
+
+- Show all details about a protocol:
+
+`birdc show protocols {{upstream1}} all`
```
Comment (blueskyson) [inline pages/common/birdc.md:25]: It would be better if you give an example value of `upstream1` in the description. I saw `all` is a valid value.

Comment (marek22k) [reply pages/common/birdc.md:25]: How exactly do you mean? `upstream1` is just a placeholder for a BGP protocol configured in bird. `all` causes not only the status but all details of the protocol to be displayed.

**Review by blueskyson — APPROVED** [review]

File: pages/common/birdc.md (hunk 3 above)
Comment (blueskyson) [reply pages/common/birdc.md:25]: Sorry, I read it wrong. Never mind.

**Pull request merged**